moped-editor/src/queries/project.js

```diff
@@ -255,13 +255,16 @@ export const TIMELINE_QUERY = gql`
         subphase_id
       }
     }
+    moped_subphases {
+      subphase_name
+      subphase_id
+    }
     moped_proj_phases(
       where: { project_id: { _eq: $projectId }, is_deleted: { _eq: false } }
-      order_by: {
-        phase_start: desc
-        moped_phase: { phase_order: desc }
-        moped_subphase: { subphase_name: asc }
-      }
+      order_by: [
```

Comment:
Thanks for mentioning the phase order in the PR description. So, will the `moped_proj_phases` table eventually have a new column to make it similar to the way milestone templates work? This is purely out of curiousity.